spec/rubocop/config_spec.rb

```diff
@@ -606,9 +606,17 @@ def cop_enabled(cop_class)
     end
 
     context 'when TargetRubyVersion is not set' do
+      let(:ruby_version_file) { '.ruby-version' }
+
+      before do
+        allow(configuration)
+          .to receive(:ruby_version_file)
+          .and_return ruby_version_file
+      end
+
       context 'when .ruby-version is present' do
         before do
-          allow(File).to receive(:file?).with('.ruby-version').and_return true
+          allow(File).to receive(:file?).with(ruby_version_file).and_return true
```

Comment:
I can't comment unmodified strings that is I put it here.
In line shoudn't `configuration.target_ruby_version` be extracted to `subject`?

There are a lot of `expect(configuration.target_ruby_version).to` which can be simplified to `is_expected` in that case. I also thought that method under `describe` should be a subject.